opentech/apply/activity/tasks.py

```diff
@@ -0,0 +1,50 @@
+from celery import Celery
+
+from django.conf import settings
+from django.core.mail import send_mail as dj_send_mail
+from django.core.mail import EmailMessage
+
+app = Celery('tasks')
+
+app.config_from_object(settings, namespace='CELERY', force=True)
+
+
+def send_mail(subject, message, from_address, recipients, log=None):
+    # Convenience method to wrap the tasks and handle the callback
+    send_mail_task.apply_async(
+        kwargs={
+            'subject': subject,
+            'body': message,
+            'from_email': from_address,
+            'to': recipients,
+        },
+        link=update_message_status.s(log.id),
+    )
+
+
+@app.task
+def send_mail_task(**kwargs):
+    response = {'status': '', 'id': None}
+    email = EmailMessage(**kwargs)
+    try:
+        email.send()
+    except Exception as e:
+        response['status'] = 'Error: ' + str(e)
```

Comment:
Could look at adding more intelligent error handling - attempt resend etc. But simple for now.